hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1236,10 +1236,17 @@
       name: chip_sw_alert_handler_lpg
       desc: '''Verify alert's low_power_group(lpg).
 
-            Choose an always-on block and send alerts during deep sleep and normal sleep mode,
-            check the alert_senders can recover after system comes out of both sleep modes.
-            Repeat the above sequence for multiple rounds with randomly chosen deep or normal sleep
-            mode for each round.
+            Three methods to check LPG:
+            - Choose an always-on or any block (for normal sleep mode only) and send alerts during
```

Comment:
Sorry, this is not clear to me. How will you send alerts if the cpu is off? Will you "force" some alert inducing action in the hardware? Also, regarding checking, since this is normal sleep I assume you intend to make sure the only interrupt you get is for wakeup, and none of the alert escalation interrupts, which I assume will be enabled.

Reply:
No worries Matute, thank you for bringing up these questions. Definitely appreciate your feedback from pwr/clk domains!

1). Agree the only way to check is via forcing. But after discussing with Tim, we think it might be more interesting to check when alert happened right before/after sleep mode. Because all I/O clocks are blocked during sleep mode, so IPs cannot really send alerts.
2). For checking - we want to make sure when resume from sleep, we can still see alerts, and if reset is issue, we shall not see any alerts.